### PR TITLE
fix: Add Eshan Gupta to the list

### LINKED ISSUE #791: Github Profile Eshan Gupta(@eshan1925)
image

### PR DESCRIPTION
**Pull request by eshan1925**
Fixes #791

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1505,4 +1505,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/60126985/138183204-9d6d4458-01e7-4394-816a-0d530a6b78ce.png",
     "issueNumber": 786
   }
```
Comment (krishguptadev) [inline data.json:1507]:
```suggestion
  },
```

File: data.json [hunk 2]
```diff
@@ -1510,5 +1510,11 @@
...
     "imageUrl": "https://user-images.githubusercontent.com/60126985/138183204-9d6d4458-01e7-4394-816a-0d530a6b78ce.png",
     "issueNumber": 786
-  }
+  },
+  {
+    "name": "Eshan Gupta",
+    "githubUsername": "eshan1925",
+    "imageUrl": "https://user-images.githubusercontent.com/78701779/138263809-68e859b3-5cb5-4e87-b006-23499ebe66e9.png",
+    "issueNumber": 791
+  },
```
Comment (krishguptadev) [inline data.json:1519]:
```suggestion
  }
```

**Review by krishguptadev — CHANGES_REQUESTED** [review]

**Comment by eshan1925** [comment]
Done @krishguptadev

File: data.json [hunk 3]
```diff
@@ -1510,5 +1510,11 @@
     "githubUsername": "fabdul88",
     "imageUrl": "https://user-images.githubusercontent.com/60126985/138183204-9d6d4458-01e7-4394-816a-0d530a6b78ce.png",
     "issueNumber": 786
+  },
+  {
+    "name": "Eshan Gupta",
+    "githubUsername": "eshan1925",
+    "imageUrl": "https://user-images.githubusercontent.com/78701779/138263809-68e859b3-5cb5-4e87-b006-23499ebe66e9.png",
```
Comment (Panquesito7) [inline data.json:1517]: I recommend more of an image like this.

image

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/51391473/138337875-218a672c-8400-4e1d-b7ef-e7ddfcb0f5e5.png",
```

**Pull request merged**